tests/bud/buildkit-mount-from/Dockerfilecachemultiplefrom

```diff
@@ -0,0 +1,4 @@
+FROM alpine
+RUN mkdir /test
+# use another image as cache source
+RUN --mount=type=cache,from=buildkitbase,target=/test --mount=type=cache,from=buildkitbase,target=/test2 cat /test2/hello
```

Comment:
This doesn't verify the contents of /test, but then `cache` mounts aren't supposed to refer to images with their `from` arguments.

Reply:
Fixed test case.